packages/block-editor/src/components/simulate-media-query/index.js

```diff
@@ -74,6 +74,10 @@ export default function useSimulatedMediaQuery( marker, width ) {
 				++ruleIndex
 			) {
 				const rule = styleSheet.cssRules[ ruleIndex ];
+				if ( rule.type !== window.CSSRule.STYLE_RULE ) {
```

Comment:
We need to add `&& rule.type !== window.CSSRule.MEDIA_RULE` because we want to catch the media rules too! The same code checks for markers and for media queries to manipulate.

Reply:
> We need to add `&& rule.type !== window.CSSRule.MEDIA_RULE` because we want to catch the media rules too! The same code checks for markers and for media queries to manipulate.

Good call. Updated in e719f4672.